Nodejs/Product/Nodejs/Extensions.cs

```diff
@@ -121,10 +121,12 @@ out project
             }
         }
 
-        internal static T[] Append<T>(this T[] list, T item) {
-            T[] res = new T[list.Length + 1];
+        internal static T[] Append<T>(this T[] list, params T[] items) {
```

Comment:
There's only one other usage of it now, and it's not perf-critical, but it doesn't hurt to make it a new one anyways.